news/2019-11-28-mappers-guild-november-newspost.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: post
+title: Featured Artist Beatmap Updates from the Mappers' Guild
+date: 2019-11-28 12:00:00 +0000
+---
+
+The Featured Artists of osu! have been receiving more love than ever before! Come and take a look at what’s been happening within the Mappers’ Guild!
+
+![](/wiki/shared/news/banners/mappersguild.png)
+
+It’s only been two months since our [previous news article](https://osu.ppy.sh/home/news/2019-09-24-mappers-guild-september-newspost) on the Mappers’ Guild, yet quite a bit has progressed with 8 quests completed and over 80 beatmaps of songs by Featured Artists have reaching ranked status! 
```

Comment:
```suggestion
It’s only been two months since our [previous news article](https://osu.ppy.sh/home/news/2019-09-24-mappers-guild-september-newspost) on the Mappers’ Guild, yet quite a bit has progressed with 8 quests completed and over 80 beatmaps of songs by Featured Artists reaching ranked status! 
```